app/Http/Requests/ChangePaswordRequest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Validation\Rules;
+
+class ChangePaswordRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'password' => ['required', 'string','confirmed', Rules\Password::defaults()],
```

Comment:
tương tự thì sửa nốt chứ =)) use ở trên cùng nhé.
```suggestion
            'password' => ['required', 'string','confirmed', Password::defaults()],
```